.gitignore

```diff
@@ -4,6 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
+.idea
```

Comment:
Seems there are some ./idea files on the repo. Most probably I would get rid of them